st2common/bin/st2ctl

```diff
@@ -69,6 +69,13 @@ function must_be_root() {
     fi
 }
 
+function not_running_in_k8s() {
+    if [ -n "$KUBERNETES_SERVICE_HOST" ]; then
+        echo -e "\e[31mError: \"st2ctl status\" is not supported under Kubernetes \e[0m\n"
```

Comment:
Maybe add something about what to use instead like "please use Kubernetes tools like kubectl to see what is running in this cluster."